docs/docs/summary_template.txt

```diff
@@ -4,6 +4,7 @@ search:
 ---
 
 - [FastStream](index.md)
+- [Getting Started](faststream.md)
```

Comment:
Probably, It should be "Features" section?

Reply:
I thought that referring to it as just features might not provide the full context for the information that this page provides, as we also explain the faststream features along with other information like install, history, and so forth. So, "Getting started" might be a better title, in my opinion. Please let me know what you think.